code/control/DeployDispatcher.php

```diff
@@ -76,31 +77,54 @@ public function history(SS_HTTPRequest $request) {
 		$start = ($page - 1) * $list->getPageLength();
 		$list->setPageStart((int) $start);
 		if (empty($list)) {
-			return $this->getAPIResponse(['message' => 'No deploy history'], 404);
+			return $this->getAPIResponse(['list' => []], 200);
 		}
 
+		$currentBuild = $this->environment->CurrentBuild();
+
 		foreach ($list as $deployment) {
 			$data[] = [
+				'ID' => $deployment->ID,
 				'CreatedDate' => $deployment->Created,
```

Comment:
Sorry, I wasn't clear. I was talking about all of the data and the format of them that we send to the front end. What is our naming convention?

upper camel case
use lower camel case
SCREAMING_SNAKE_CASE?